src/Options.js

```diff
@@ -158,7 +183,7 @@ class Options {
     /**
      * Create a new options objects with the give font.
      */
-    withFont(font) {
+    withFont(font: ?string): Options {
         return this.extend({
             font: font || this.font,
```

Comment:
All b/c we use `"\\text": undefined` in `textFunctionFonts`.  That's something we should maybe look at changing in the future.